specification/resources/resource-manager/Microsoft.Resources/preview/2023-07-01-preview/changes.json

```diff
@@ -0,0 +1,496 @@
+                "value": "Delete",
+                "description": "An existing resource was deleted",
+                "name": "Delete"
+              },
+              {
+                "value": "Create",
+                "description": "A newly created resource",
+                "name": "Create"
+              }
+            ]
+          }
+        },
+        "changeAttributes": {
+          "$ref": "#/definitions/ChangeAttributes"
+        },
+        "changes": {
+          "$ref": "#/definitions/ChangesDictionary"
+        }
+      }
+    },
+    "ChangeAttributes": {
+      "description": "Details about the change resource",
+      "type": "object",
+      "properties": {
+        "correlationId": {
+          "description": "The ARM correlation ID of the change resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "timestamp": {
+          "description": "The time the change(s) on the target resource ocurred",
+          "type": "string",
+          "readOnly": true
+        },
+        "changesCount": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The number of changes this resource captures"
+        },
+        "previousResourceSnapshotId": {
+          "description": "The GUID of the previous snapshot",
+          "type": "string",
+          "readOnly": true
+        },
+        "newResourceSnapshotId": {
+          "description": "The GUID of the new snapshot",
+          "type": "string",
+          "readOnly": true
+        },
+        "isTruncated": {
+          "description": "The flag indicating whether property changes dictionary was truncated in this resource",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "changedBy": {
+          "description": "The entity that made the change(s)",
+          "type": "string",
+          "readOnly": true
+        },
+        "changedByType": {
+          "description": "The entity type that made the change(s)",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Application",
+            "User",
+            "ManagedIdentity"
+          ],
+          "x-ms-enum": {
+            "name": "changedByType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Application",
+                "description": "An application made the change(s)",
+                "name": "Application"
+              },
+              {
+                "value": "User",
+                "description": "A user made the change(s)",
+                "name": "User"
+              },
+              {
+                "value": "ManagedIdentity",
+                "description": "A ManagedIdentity made the change(s)",
+                "name": "ManagedIdentity"
+              }
+            ]
+          }
+        },
+        "clientType": {
+          "description": "How the change was made",
```

Comment:
>How the change was made", [](http://example.com/codeflow?start=26&length=25)

something better, since it goes into docs
*Client used to make the changes* or better